test/e2e/persistent_volumes.go

```diff
@@ -555,7 +560,7 @@ func makeWritePod(ns string, pvcName string) *api.Pod {
 					Name:    "write-pod",
 					Image:   "gcr.io/google_containers/busybox:1.24",
 					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "touch /mnt/SUCCESS && exit 0 || exit 1"},
+					Args:    []string{"-c", "touch /mnt/SUCCESS && (id -G | grep '^777 \\| 777 \\| 777$') && exit 0 || exit 1"},
```

Comment:
* that won't catch '^777$'
* does it really work? `echo '666 777' | grep '^777 | 777 | 777$` does not match 777, `egrep` does.

what about PCRE? `grep -P '\b777\b'` probably matches everything,